src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Random randomNumber = new Random();
```

Comment:
Consider making `randomNumber` a private static field to restrict its access to within the `Lottery` class only, following the principle of encapsulation.